codeboarding_workflows/sources/remote.py

```
@@ -0,0 +1,97 @@
+from codeboarding_workflows.full import generate_analysis
+from codeboarding_workflows.markdown import generate_markdown_docs
+from diagram_analysis import RunContext
+from repo_utils import clone_repository, get_repo_name, upload_onboarding_materials
+from utils import create_temp_repo_folder, remove_temp_repo_folder
+
+logger = logging.getLogger(__name__)
+
+
+def onboarding_materials_exist(project_name: str) -> bool:
+    generated_repo_url = f"https://github.com/CodeBoarding/GeneratedOnBoardings/tree/main/{project_name}"
+    response = requests.get(generated_repo_url)
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)